pints/_abc/_abc_rejection.py

```diff
@@ -70,7 +70,7 @@ def tell(self, fx):
         if np.sum(accepted) == 0:
             return None
         else:
-            return [self._xs.tolist() for c, x in
+            return [x.tolist() for c, x in
```

Comment:
What is the "c" doing here? It looks like it isn't used within the list comprehension...?

Reply:
`c` was the index, so for `accepted = [0.1, 0.2]`, we would have c=0, x=0.1 and c=1, x=0.2. I think this is not needed, I will simply removed the usage of enumerate.